cluster.go

```diff
@@ -1182,6 +1182,19 @@ func (c *ClusterClient) reaper(idleCheckFrequency time.Duration) {
 	}
 }
 
+// Pipeline returns mechanism to realise Redis Pipeline technique.
+//
+// Pipelining is a technique to extremely speed up processing by packing
+// operations to batches, send them at once to Redis and read a replies in a
+// singe step.
+// See https://redis.io/topics/pipelining
+//
+// Pay attention, that Pipeline is not a transaction, so you can get unexpected
+// results in case of big pipelines and small read/write timeouts.
+// Redis client has retransmission logic in case of timeouts, pipeline
+// can be retransmitted and commands can be executed more then once.
+// To avoid this: it is good idea to use reasonable bigger read/write timeouts
+// depends of your batch size and/or use TxPipeline.
 func (c *ClusterClient) Pipeline() Pipeliner {
```

Comment:
Could you please move this comment to the `Pipeliner` interface so we have 1 copy of this text?

Reply:
Done.
But idea to place this comment with Pipeline function base on that: mostly users call this function to get Pipeliner, but not got directly Pipeliner interface.
So it is logically closer to Pipeline function.